blocks/api/raw-handling/test/inline-block-corrector.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * External dependencies
+ */
+import { equal } from 'assert';
+
+/**
+ * Internal dependencies
+ */
+import inlineBlockCorrector from '../inline-block-corrector';
+import { deepFilterHTML } from '../utils';
+
+describe( 'inlineContentConverter', () => {
+	it( 'should move inline-block content from paragraph', () => {
```

Comment:
It would be useful to have some of these behaviors documented (`README.md` and/or JSDoc on the functions themselves). By the name `inlineBlockCorrector`, I had no idea what to expect from its behavior.

I'm coming to this mostly unfamiliar with the raw handlers, so I expect it'd be a common sentiment.